### ISSUE TITLE
Some sort of minimal Address book feature

### ISSUE THREAD

**Issue by nezam05** [ramiroaisen/raven-webmail#52]
To make this usable on Production, a minimal address book feature with auto suggesting email address during mail compose would be nice.

**Comment by ramiroaisen** [comment]
Yes, this Is my top priority for next features, It Is doable, wildduck exposes an api endpoint for that. Any help will be welcomed
